projecttacoma/measure-repository-service-test-kit
# Measure $data-requirements suite

# Summary
¡NEW Measure $data-requirements test suite!

## New behavior
There's a whole new $data-requirements test suite for Measures now

## Code changes
 - created `measure_data_requirements.rb` file containing measure_data_requirements suite
 - added `assertion_utils.rb` for ease or repeated assertions
 - added testing for new suite

# Testing guidance
 - `rspec`
 - Start up the test kit with the instruction in the readme
 - In `measure-repository-service`
 - `npm run db:reset`
 - `npm run db:loadBundle "ecqm-content-r4-2021/bundles/measure/ColorectalCancerScreeningsFHIR/ColorectalCancerScreeningsFHIR-bundle.json"`
 - `npm start`
 - Navigate to `http://localhost:4567`
 - Select the Measure $data-requirements suite
 - Click "run all tests"
 - Enter the following information for each field: 
 - Measure id: `ColorectalCancerScreeningsFHIR`
 - FHIR Server Base: `http://localhost:3000`
 - Measure url: `http://ecqi.healthit.gov/ecqms/Measure/ColorectalCancerScreeningsFHIR`
 - Measure version: `0.0.003`
 - Measure Identifier: `http://hl7.org/fhir/cqi/ecqm/Measure/Identifier/cms|130FHIR`
 - Make sure all tests pass!
